communication/src/actor/ask.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020-2021 IOTA Stiftung
```

Comment:
Maybe its worth consolidating these files into a library called utils since this code appears in the interface as well.

Reply:
We already have the `test_utils` library, maybe we could rename it to `utils` and have separate modules in it for `test` stuff and `actor` or something? Or is it better to separate it completely

Reply:
Hmm, perhaps it could be done that way.